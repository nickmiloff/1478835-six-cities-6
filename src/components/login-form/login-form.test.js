import React from 'react';
import '@testing-library/jest-dom';
import {render} from '@testing-library/react';
import {TestStore as Test} from '../test/test';

import LoginForm from './login-form';

describe(`LoginForm component test`, () => {
  it(`LoginForm component' should render correctly`, () => {
    const {container} = render(
        <Test>
          <LoginForm />
        </Test>
    );

    expect(container).toMatchSnapshot();
  });
});
