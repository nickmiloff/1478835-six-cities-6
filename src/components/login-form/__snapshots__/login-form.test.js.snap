// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginForm component test LoginForm component' should render correctly 1`] = `
<div>
  <form
    action="#"
    class="login__form form"
    data-testid="form"
    method="post"
  >
    <div
      class="login__input-wrapper form__input-wrapper"
    >
      <label
        class="visually-hidden"
      >
        E-mail
      </label>
      <input
        autocomplete="on"
        class="login__input form__input"
        data-testid="email"
        name="email"
        placeholder="Email"
        required=""
        type="email"
        value=""
      />
    </div>
    <div
      class="login__input-wrapper form__input-wrapper"
    >
      <label
        class="visually-hidden"
      >
        Password
      </label>
      <input
        autocomplete="on"
        class="login__input form__input"
        data-testid="password"
        name="password"
        placeholder="Password"
        required=""
        type="password"
        value=""
      />
    </div>
    <button
      class="login__submit form__submit button"
      type="submit"
    >
      Sign in
    </button>
  </form>
</div>
`;
